openshift/templates/pims-api/deploy.yaml

```diff
@@ -18,34 +20,65 @@ objects:
   - kind: Route
     apiVersion: v1
     metadata:
-      name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}
+      name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
+      namespace: ${PROJECT_NAMESPACE}-${ENV_NAME}
       labels:
-        name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}
+        name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
         app: ${APP_NAME}
         component: ${COMP_NAME}
         env: ${ENV_NAME}
         role: backend
+        id: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID}
```

Comment:
What is this giving us that the three instance of name: ${APP_NAME}-${COMP_NAME}-${ENV_NAME}${ID} aren't?

Reply:
I've updated to use the `name` label instead.